app/models/rule.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Rule < ApplicationRecord
```

Comment:
Could this use a `has_many :caution_flag_rules` ?

Reply:
It's a 1 to 1 relationship at the moment, i have a `belongs_to` on CautionFlagRules to allow for use of `priority` of a Rule when retrieving a CautionFlagRule so that i can apply rules in a specific order